packages/server/lib/browsers/utils.ts

```diff
@@ -398,46 +398,99 @@ const throwBrowserNotFound = function (browserName, browsers: FoundBrowser[] = [
 // Chromium browsers and webkit do not give us pre requests for download links but they still go through the proxy.
```

Comment:
Seems like this comment should be updated.

Reply:
Update: https://github.com/cypress-io/cypress/pull/28517/commits/956dd0aa92c8e7cdb0ee1284be92bdeeb52426a3